packages/core/src/api/decorators/transaction.decorator.ts

```diff
@@ -31,6 +31,17 @@ export const TRANSACTION_MODE_METADATA_KEY = '__transaction_mode__';
  */
 export type TransactionMode = 'auto' | 'manual';
 
+export const TRANSACTION_ISOLATION_LEVEL_METADATA_KEY = '__transaction_isolation_level__';
+/**
+ * @description
+ * Transactions can be run at different isolation levels. The default is "SERIALIZABLE", which
+ * is the only correct mode. See the documentation of your database for more information on available
+ * isolation levels.
+ * 
+ * @default 'SERIALIZABLE'
```

Comment:
Can you update the description to say that it defaults `undefined` which will fall back to the default of your database. Also add the following tags.

```
 * @docsCategory request
 * @docsPage Transaction Decorator
```

so that it appears in the documentation website on the correct page